src/adaptors/adaptor_common.c

```diff
@@ -66,7 +66,8 @@ qd_error_t qd_load_adaptor_config(qdr_core_t *core, qd_adaptor_config_t *config,
     config->site_id = qd_entity_opt_string(entity, "siteId", 0);               CHECK();
     config->ssl_profile_name  = qd_entity_opt_string(entity, "sslProfile", 0); CHECK();
     config->authenticate_peer = qd_entity_opt_bool(entity, "authenticatePeer", false); CHECK();
-    config->verify_host_name  = qd_entity_opt_bool(entity, "verifyHostname", false);   CHECK();
+    config->verify_host_name  = qd_entity_opt_bool(entity, "verifyHostname", false);
+    CHECK();
```

Comment:
Is clang format causing this diff to appear ? Can you simply restore this file to its original version and not run the clang formatter ?